### ISSUE TITLE
Should Chapters Raise Money Only for Themselves, or Collectively/Globally??

### ISSUE THREAD

**Issue by BethanyG** [pyladies/project-finance#7]
Should we have chapters decide on fundraising individually? Or focus on a central collection?

**Comment by MmeMarieLouise** [comment]
@BethanyG what do you think ?

**Comment by BethanyG** [comment]
I agree with @econchick  -- chapters should be free (_and willing, with our support_) to fundraise and also ask for in-kind donations of space or other items.  They should also be able to offer specific "trades" of sponsorship -- e.g., having signage at meetings/trainings that says "This was sponsored by ______" -- within limits.  We need to outline what the rules would be for those kinds of things -- and  we'd need to really research what is and is not allowed under the [501(c)(3)]() rules.

Ideally, every active and verified chapter would have an account that was "seeded" annually with a specific "support" amount from the central PyLadies "pot" -- and then they'd be free to add to it with whatever fundraising they're willing/able to do...and administer the funds the way they see fit (_provided, of course that they use them in line with the PSF/PyLadies mission, report on their expenditures and activities, etc._).  It would be expected that they'd contribute to/help with global fundraising efforts.  But beyond that global community obligation, they could then fundraise toward their own projects/chapters.

Unfortunately, I think that we can't set things up like that.  The PSF doesn't have the bandwidth to administer individual accounts for all chapters, and its not at all clear how we'd make that work on an international level -- the non-profit/tax exempt status is specific to the US, and then there are banking and pricing and governance considerations.

I think earmarking within a central account makes sense -- provided that it doesn't over-stress the PSF staff, and that we're public and transparent about how it works, and what each chapter is spending/receiving .  The only thing we'd need to figure out is what we do with NY and Chicago and any other chapters (Vancouver??) that already have their own accounts and administration.  We may need to leave those as-is.  

I also think we should think of an annual support amount for each active & verified chapter.  Don't have a solid proposal at the moment, but my though would be that each chapter could follow a streamlined expensing process, as long as they were under their annual support limit (or they had earmarked funds they'd raised themselves). That would give them autonomy to spend on things that they felt were needed or important (_but, as always they'd need to be applicable to the organizations mission_), rather than the GC making decisions on categories of services or expenses that we will or will not pay for.  Anything beyond the earmarks would require a more formal proposal and review.  Big expenses or projects would require a full grant proposal GC review and PSF review.
